code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -267,7 +267,7 @@
 		return
 	if(!item_to_add)
 		user.visible_message("[user] pets [src].","<span class='notice'>You rest your hand on [src]'s head for a moment.</span>")
-		SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "pet_corgi", /datum/mood_event/pet_corgi)
+		SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "pet_doggie", /datum/mood_event/pet_doggie, initial(src.name))
```

Comment:
Ian's initial name is Ian.

Reply:
Ians are adorable!

Reply:
Yeah I realized that while working on the cat pr, I'm gonna have to get creative with the wording. Maybe check if the first letter is capitalized?

Reply:
you could just make it singular lol

`description = "<span class='nicegreen'>\The [name] is adorable! I can't stop petting \him!</span>\n"`

Reply:
but that would be admitting i couldn't make the capitalize proc work...

well alright fine